app/frontend/actions.js

```diff
@@ -103,7 +100,10 @@ module.exports = ({setState, getState}) => {
       const sendAddress = {fieldValue: ''}
       const sendResponse = ''
       const usingTrezor = cryptoProvider === 'trezor'
-      const isDemoWallet = secret === ADALITE_CONFIG.ADALITE_DEMO_WALLET_MNEMONIC
+      const isDemoWallet =
+        walletSecretDef && // we can also use !usingTrezor
```

Comment:
I don't understand the "we can also use !usingTrezor" comment - either remove it or  provide a more elaborate explanation

Reply:
When we merge it with ledger integration, there will be usingHwWallet boolean. In my opinion, using this boolean would be more descriptive than checking just if walletSecretDef is present.